### PR TITLE
Overflow issue fix plus couple of improvment 

### PR DESCRIPTION
**Pull request by medyas**
fixed overflow when using more the 7 pages - upgraded flutter_svg - added support for custom skip and done button color/textStyle/shape
#16 #15

### CODE REVIEW

**Comment by xsahil03x** [comment]
@medyas thanks for opening a pull request, I'll soon review and merge it 👍

**Comment by medyas** [comment]
@xsahil03x thanks, and can you inform when its published to pub.dev because i am using the repo branch in my app.

File: pubspec.yaml [hunk 1]
```diff
@@ -7,13 +7,13 @@ homepage: https://github.com/xsahil03x/fancy_on_boarding
...
-  flutter_svg: ^0.15.0
+  flutter_svg: ^0.17.2
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.7.0 <3.0.0"
```
Comment (xsahil03x) [inline pubspec.yaml]: Can you revert this to 2.1.0 as we are not using any new SDK features yet?

Comment (medyas) [reply pubspec.yaml]: done

**Comment by xsahil03x** [comment]
Thank you @medyas for the PR. Cheers :+1:

**Pull request merged**